docs/TuxStores.md

```diff
@@ -0,0 +1,182 @@
+
+```
+    store.emitChange(EVENT_TYPE); // OR
+    store.emitChange(); // will emit the default change event type string: 'CHANGE'
+```
+
+##### <a id="emitChange-EVENT_TYPE"></a>Parameter - `EVENT_TYPE` - type: STRING - optional [#](#emitChange-EVENT_TYPE)
+Change event type string that will be broadcast for any listeners that are listening for that event type. If no `EVENT_TYPE` is specified the default value of `CHANGE` will be used.
+
+```
+    var newEventType = 'NEW_EVENT_TYPE';
+    store.emitChange(newEventType);
+```
+
+### <a id="store-addChangeListener"></a>Method `store.addChangeListener` - type: FUNCTION [#](#store-addChangeListener)
+The `addChangListener` method will invoke the provided callback function when the specified event type string is emitted. The change event can be either the default event string `CHANGE` or one provided by the user.
+
+```
+    store.addChangeListener(callback, EVENT_TYPE); // OR
+    store.addChangeListener(callback); // this will add a listener for the default CHANGE event
+```
+
+##### <a id="addChangeListener-callback"></a>Parameter - `callback` - type: FUNCTION - required [#](#addChangeListener-callback)
+The callback to invoke when corresponding event is triggered.
+
+```
+    var listenerCallback = function () {
+      this.setState({ messages: MessageStore.all() });
+    };
+
+    store.addChangeListener(listenerCallback);
+```
+
+##### <a id="addChangeListener-EVENT_TYPE"></a>Parameter - `EVENT_TYPE` - type: STRING - optional [#](#addChangeListener-EVENT_TYPE)
+Event string to add a listener callback to. If no event is passed in than the listener will be added to the default `CHANGE` event.
+
+```
+    var listenerCallback = function () {
+      this.setState({ messages: MessageStore.all() });
+    };
+
+    var newEventType = 'NEW_EVENT_TYPE';
+    store.addChangeListener(listenerCallback, newEventType);
+```
+
+
+### <a id="store-removeChangeListener"></a>Method `store.removeChangeListener` - type: FUNCTION [#](#store-removeChangeListener)
+Removes the passed in listener callback from the passed in event string. If no event is specified than the default `CHANGE` event is used.
+
+```
+    store.removeChangeListener(callback, EVENT_TYPE);
+```
+
+##### <a id="removeChangeListener-callback"></a>Parameter - `callback` - type: FUNCTION - required [#](#removeChangeListener-callback)
+The callback function that will be removed from the stores set of listeners.
+
+```
+    var listenerCallback = function () {
+      this.setState({ messages: MessageStore.all() });
+    };
+
+    store.addChangeListener(listenerCallback);
+    store.removeChangeListener(listenerCallback);
+```
+
+##### <a id="removeChangeListener-EVENT_TYPE"></a>Parameter - `EVENT_TYPE` - type: STRING - optional [#](#removeChangeListener-EVENT_TYPE)
+Change event type string that will be used to remove the listener from the store. If no `EVENT_TYPE` is specified the default value of `CHANGE` will be used.
```

Comment:
Change to Change event type string on the store that the listener will be removed from.